src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,59 @@
 package core.basesyntax;
 
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
 public class WorkWithFile {
```

Comment:
Pay attention to the access modifiers. Methods that are only intended to be used within the same class should not be `public`. For example, `readDataFromFile`, `createReport`, `writeDataToFile`, and `resetCalculations` could be private if they are only called from within this class.